.coafile

```diff
@@ -29,6 +29,8 @@ bears = SpaceConsistencyBear
 
 [commit]
 bears = GitCommitBear
+host_used = github
+contains_url = True
```

Comment:
I'm reverting the `.coafile` back to normal anyway @jayvdb 
But, do suggest me how to differentiate between issue related commits and non issue related ones?